### PR TITLE
SDI-4: ✨ Minor ui improvements

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: backend/controllers/prisonerProfile/prisonerVisits.ts [hunk 1]
```diff
@@ -157,8 +157,9 @@ export default ({ prisonApi, pageSize = 20 }) =>
         results,
         visitTypes: VISIT_TYPES,
         filterApplied: Boolean(fromDate || toDate || visitType),
-        prisons: hasLength(prisons) && prisons.map((type) => ({ value: type.prisonId, text: type.prison })),
```
Comment (petergphillips) [inline backend/controllers/prisonerProfile/prisonerVisits.ts:160]: this was causing `false` to be sent to nunjucks which then didn't add the `All` option

File: backend/controllers/prisonerProfile/prisonerVisits.ts [hunk 2]
```diff
@@ -157,8 +157,9 @@ export default ({ prisonApi, pageSize = 20 }) =>
...
+        prisons: hasLength(prisons) ? prisons.map((type) => ({ value: type.prisonId, text: type.prison })) : [],
         statuses,
+        profileUrl: `/prisoner/${offenderNo}`,
```
Comment (petergphillips) [inline backend/controllers/prisonerProfile/prisonerVisits.ts:162]: missing link for going back

File: integration-tests/plugins/index.js [hunk 3]
```diff
@@ -28,6 +28,7 @@ const extractOffenderNumbers = (activityList) => {
 
 module.exports = (on) => {
   on('task', {
+    ...prisonApi,
```
Comment (petergphillips) [inline integration-tests/plugins/index.js:31]: just a bit of tidy up by importing all from prison api

File: views/prisonerProfile/prisonerVisits/partials/prisonerVisitsSearchForm.njk [hunk 4]
```diff
@@ -55,6 +47,14 @@
         attributes: { 'data-test': 'status-type' }
       }) }}
 
+      {{ govukSelect({
```
Comment (petergphillips) [inline views/prisonerProfile/prisonerVisits/partials/prisonerVisitsSearchForm.njk:50]: was in the wrong place - should be after the status filter

**Review by hyderkhanmoj — APPROVED** [review]
LGTM

**Pull request merged**